designer/src/components/section-editor.tsx

```diff
@@ -67,7 +67,7 @@ export const SectionEditor = ({
   );
   const dispatch = useAppDispatch();
 
-  console.log('SectionEditor', viewId, viewSet);
+  console.log('SectionEditor', { viewId, viewSet, fView });
```

Comment:
At some point we should remove this debug output but since it's there for all the main components we'll leave it in for now.